create-test-data.py

```diff
@@ -713,6 +739,8 @@ def put_entry(self, entry):
 @click.option("--with-hbac", default=False, help="Create hbac rules.",
               is_flag=True)
 @click.option("--debug", default=False, help="Debug logging", is_flag=True)
+@click.option("--number-of-subgroups", default=0, help="Number of subgroups to create.",
+              type=int)
```

Comment:
There's no need to specify the type if there's a default value.

Reply:
removed